what_vpn/requests.py

```diff
@@ -18,10 +19,13 @@ def __init__(self, ssl_version=None, **kwargs):
         super().__init__(**kwargs)
 
     def init_poolmanager(self, connections, maxsize, block=False):
+        ssl_context = ssl.SSLContext(self.ssl_version)
+        ssl_context.set_ciphers('DEFAULT@SECLEVEL=1')
```

Comment:
I think we should do `@SECLEVEL=0` here. https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_security_level.html

Reply:
I believe `@SECLEVEL=1` is sufficient nowadays, but indeed if users are OK with lower security, just go all the way to 0.